quinn-proto/src/connection/mod.rs

```diff
@@ -643,6 +656,7 @@ where
                 (num_datagrams - 1) * (self.path.mtu as usize),
                 ack_eliciting,
                 self,
+                *self.supported_versions.start(),
```

Comment:
I think we should probably also make the configuration about which is the explicit/initiating version.